pytds/__init__.py

```diff
@@ -631,6 +631,8 @@ def execute(self, operation, params=()):
         """
         conn = self._assert_open()
         conn._try_activate_cursor(self)
+        if operation[:3] == "sp_":
```

Comment:
that is not good I think, why not use standard callproc method: http://python-tds.readthedocs.org/en/latest/pytds.html#pytds.Cursor.callproc,
https://www.python.org/dev/peps/pep-0249/#callproc
?